data/avro-schema/src/main/resources/avro/net/corda/data/p2p/gateway/LinkManagerResponse.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "record",
+  "name": "LinkManagerResponse",
+  "namespace": "net.corda.data.p2p.gateway",
+  "doc": "A response from the p2p link manager to the p2p gateway component",
```

Comment:
similar spirit, might be worth making it a bit more generic (even though it's only used by the gateway currently) e.g.
```
"doc": "A response from the p2p link manager when being sent a LinkInMessage via the HTTP endpoint",
```